package.json

```diff
@@ -20,14 +20,14 @@
     "watch-js-development": "webpack --config config/webpack.config.js --watch",
     "build-js-production": "cross-env NODE_ENV=production webpack --config config/webpack.config.js",
     "watch-js-production": "cross-env NODE_ENV=production webpack --config config/webpack.config.js --watch",
-    "build-images-development": "cpx \"src/img/**\" dist/img --verbose",
-    "watch-images-development": "cpx \"src/img/**\" dist/img --watch --verbose",
-    "build-images-production": "cpx \"src/img/**\" dist/img --verbose",
-    "watch-images-production": "cpx \"src/img/**\" dist/img --watch --verbose",
-    "build-all-development": "npm run clean && run-p -l build-html-development build-css-development build-js-development build-images-development build-fonts",
-    "watch-all-development": "npm run clean && run-p -l watch-html-development watch-css-development watch-js-development watch-images-development watch-fonts",
-    "build-all-production": "npm run clean && run-p -l build-html-production build-css-production build-js-production build-images-production build-fonts",
-    "watch-all-production": "npm run clean && run-p -l watch-html-production watch-css-production watch-js-production watch-images-production watch-fonts",
+    "build-other": "cpx \"WebContent/**/*.txt\" dist --verbose",
```

Comment:
Need to merge `develop` and correct these four NPM scripts to point to the `src` directory instead of `WebContent` directory.